check_stability.py

```diff
@@ -283,6 +286,17 @@ def wptrunner_args(self, root):
             "test_types": ["testharness", "reftest"]
         }
 
+    def extra_setup(self):
+        # https://bugs.chromium.org/p/chromium/issues/detail?id=713947
+        logger.debug("DBUS_SESSION_BUS_ADDRESS %s" % os.environ.get("DBUS_SESSION_BUS_ADDRESS"))
+        if "DBUS_SESSION_BUS_ADDRESS" not in os.environ:
+            if find_executable("dbus-launch"):
+                logger.debug("Attempting to start dbus")
+                logger.debug(subprocess.check_output(["dbus-launch"]))
+            else:
+                logger.debug("Setting DBUS_SESSION_BUS_ADDRESS to a placeholder value")
+                os.environ["DBUS_SESSION_BUS_ADDRESS"] = "/dev/null"
```

Comment:
I don't currently understand the implications of using this value. I am going to take some time to learn what this actually means.